README.md

```diff
@@ -1,24 +1,112 @@
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Blog App <a name="about-project"></a>
 
-* System dependencies
+**Blog App** is a fully functional web platform that allows users to create and manage their blog posts, interact with content by commenting and liking posts, and engage in a vibrant online community. This project follows a structured development process through multiple milestones, including data model creation, validations, controllers, views, authentication, authorization, and API endpoints.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
+Ruby on Rails, Visual Studia Code, WSL
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+- **Ruby on Rails**
 
-* How to run the test suite
+- **WSL**
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Blog Post Management**
+- **User Profiles**
+- **Authorization**
+- **API Endpoints**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+In order to use this project.. Clone this repository to your desired folder by pasting this command in your command line interface:
+
+  https://github.com/Kaiserabbas/blog-app.git
+
+### Prerequisites <a name="prerequisites"></a>
+
+  Ruby on Rails
+  Git
+
+### Install <a name="install"></a>
+
+To install the project's dependencies, run:
+
+```
+bundle install
+```
+
+### Run tests <a name="run tests"></a>
+
+To test, run:
+
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+- GitHub: [@Qaisar](https://github.com/Kaiserabbas)
+- LinkedIn: [Qaisar](https://www.linkedin.com/in/qaisar-abbas-21a93840/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Search Functionality**
+- [ ] **User Notifications**
+- [ ] **Social Media Integration**
+- [ ] **Mobile App**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Kindly give this project a star if you like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) license.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I also noticed your MIT link is broken so kindly make sure to fix this as it makes your repo look unprofessional 👶

Reply:
Licence file is created and link has been updated. Thank You